### PR TITLE
6.get/api/articles/ article id/comments

### PR DESCRIPTION
**Pull request by kyron321**

### CODE REVIEW

File: __tests__/articles.test.js [hunk 1]
```diff
@@ -90,3 +90,62 @@ describe("GET/api/articles/:article_id", () => {
...
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
```
Comment (robert112233) [inline __tests__/articles.test.js:129]: Lines 121-129 - This is just a repeat of the test on line 95 right? Is it necessary to repeat yourself?

File: __tests__/articles.test.js [hunk 2]
```diff
@@ -90,3 +90,62 @@ describe("GET/api/articles/:article_id", () => {
...
+
+  test("responds with a 400 error when passed an article id formatted incorrectly", () => {
+    return request(app)
+      .get("/api/articles/incorrectformat/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+});
```
Comment (robert112233) [inline __tests__/articles.test.js:151]: Missing 200 test - When requesting from an article that has no comments, should return an empty array 👍

File: controllers/controllers.app.js [hunk 3]
```diff
@@ -28,3 +29,12 @@ exports.getArticlesByArticleID = (req, res, next) => {
     })
     .catch(next);
 };
+
```
Comment (robert112233) [inline controllers/controllers.app.js:32]: Here you may need a checkIfArticleExists function to differentiate between a valid article with no comments (200), and a non existent article (404) . Both of these will return an empty array but need to be handled differently

**Review by robert112233 — CHANGES_REQUESTED** [review]
Great work so far, see comments and submit another PR for review 👍
